### PR TITLE
Rework the make-* scripts for simpler toolchain

### PR DESCRIPTION
**Pull request by josuah**
Each commit address an independent point, with the global goal of making the toolchain simpler, and easier to rework with further on: #28, #2, #35, #107

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -908,25 +899,19 @@ warn-auto.sh mailsubj.sh conf-qmail conf-break conf-split
...
-make-compile: \
-make-compile.sh auto-ccld.sh
-	cat auto-ccld.sh make-compile.sh > make-compile
-	chmod 755 make-compile
-
-make-load: \
-make-load.sh auto-ccld.sh
-	cat auto-ccld.sh make-load.sh > make-load
-	chmod 755 make-load
+compile: \
```
Comment (DerDakon) [inline Makefile:902]: Please keep the alphabetical order of the Makefile targets.

File: elq.sh [hunk 2]
```diff
@@ -1 +1 @@
-QMAIL/bin/maildir2mbox && exec elm ${1+"$@"}
```
Comment (DerDakon) [inline elq.sh:1]: Don't touch elq, it will die right after 1.08. Same for pinq and qail.

File: makelib.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+main="$1"; shift
```
Comment (DerDakon) [inline makelib.sh:1]: "main" isn't really needed, one could use $1 at the 2 places where it matters below and just $* for the AR line.

Comment (josuah) [reply makelib.sh]: I replicated the previous scripts directly, I prefer as you suggest.

File: Makefile (hunk 1 above)
Comment (josuah) [reply Makefile]: I am now learning this best practice for large makefiles.

File: elq.sh (hunk 2 above)
Comment (josuah) [reply elq.sh]: I applied the change to all except this one.

**Comment by josuah** [comment]
FYI: Just rebased it to master.

**Pull request closed without merging**